### PR TITLE
refresh reviews page (bug 1118631)

### PR DESCRIPTION
**Pull request by ngokevin**
- in the shots, there's a misalignment with the timestamp. that's been fixed.

screen shot 2015-01-08 at 3 05 22 pm
another screenshot
screen shot 2015-01-08 at 3 05 26 pm
screen shot 2015-01-08 at 3 06 34 pm

### CODE REVIEW

**Comment by ngokevin** [comment]
@pwalm maybe we should move the header out of the main section like we have the New and Popular pages?

**Comment by pwalm** [comment]
Hmm... Let's leave it as is. We could centre "Reviews" on mobile though. But I wouldn't worry about it.

**Comment by chuckharmston** [comment]
Idle observation: we could use some `line-height` on the review text.

**Comment by ngokevin** [comment]
@pwalm typo for reviews? do i use the ```body``` type as listed in http://marketplaceux.github.io/marketplace-style-guides/type.html

**Comment by pwalm** [comment]
yep, but use #1a1a1a for the type colour.

**Comment by ngokevin** [comment]
Looks way better. Need to get this style implemented to some other areas

screen shot 2015-01-08 at 4 31 47 pm

**Comment by ngokevin** [comment]
also not sure if we need the exact time of the review...date should suffice, maybe shrink it enough to inline it with the review actions

**Comment by pwalm** [comment]
No time, just date. You're right, it's irrelevant info.

—Philip Walmsley

Senior Visual Designer @ Mozilla




m: 604-339-5941


t: @pwalmsley

**Comment by spasovski** [comment]
Best dating advice I've heard in a while.

**Comment by pwalm** [comment]
lol

—Philip Walmsley

Senior Visual Designer @ Mozilla




m: 604-339-5941


t: @pwalmsley

**Comment by ngokevin** [comment]
timestamp rmed

screen shot 2015-01-09 at 9 51 25 am

File: src/media/css/reviews.styl [hunk 1]
```diff
@@ -336,6 +246,9 @@ mkt-prompt[data-modal="flag-review"] {
     .review .stars {
         right: 20px;
     }
+    .reviews-listing h3{
```
Comment (spasovski) [inline src/media/css/reviews.styl]: `h3 {`

File: src/media/js/utils_local.js [hunk 2]
```diff
@@ -71,6 +71,8 @@ define('utils_local',
         // `navigator.onLine` is unreliable (see bugs 654579 and 756364)
         // so we need to test an actual connection.
         var def = defer.Deferred();
+        online(def);
```
Comment (ngokevin) [inline src/media/js/utils_local.js]: oops!

File: src/media/js/helpers_local.js [hunk 3]
```diff
@@ -22,6 +22,10 @@ define('helpers_local',
     ];
 
     /* Register filters. */
+    filters.date = function(date) {
+        return new Date(date).toLocaleDateString();
```
Comment (spasovski) [inline src/media/js/helpers_local.js]: When will humanity learn to stick to the Mayan calendar?

File: src/media/js/ratings.js [hunk 4]
```diff
@@ -121,10 +121,12 @@ define('ratings',
...
                     user: 'mine'
                 });
-                $('.review-button').html(gettext('Loading...'));
+                this.innerHTML = gettext('Loading...');
+
+                var self = this;
                 requests.get(endpoint).done(function(existingReview) {
                     initReviewModal(existingReview.objects[0]);
-                    $('.review-button').html(gettext('Edit your Review'));
+                    self.innerHTML = self.getAttribute('data-text');
```
Comment (spasovski) [inline src/media/js/ratings.js]: Any reason for avoiding jQuery? Not a complaint just wondering.

Comment (ngokevin) [inline src/media/js/ratings.js]: there's no reason to use jquery. it'd be less performant

File: src/templates/ratings/rating.html [hunk 5]
```diff
@@ -7,7 +7,7 @@
   {% defer (url=api('review', [id])) %}
     <div class="reviews reviews-listing single-review">
       <ul class="ratings-placeholder-inner">
-        {{ rating(this) }}
+        {{ review(this) }}
```
Comment (spasovski) [inline src/templates/ratings/rating.html]: We had switched to "ratings" from "reviews" at one point but I don't remember the full story behind it. Doesn't really matter I agree with keeping it consistent and I noticed you changed the file names too which is great.

**Comment by spasovski** [comment]
Looks great locally r+wc

File: src/templates/ratings/rating.html (hunk 5 above)
Comment (ngokevin) [inline src/templates/ratings/rating.html]: clashes with content ratings

File: src/media/js/helpers_local.js (hunk 3 above)
Comment (ngokevin) [inline src/media/js/helpers_local.js]: my mac clock is a unix timestamp

**Comment by ngokevin** [comment]
thanks for taking time out of the preview work for this

**Pull request merged**